packages/java-shell/src/test/resources/literal/long.expected.2.txt

```diff
@@ -0,0 +1,3 @@
+LongResult: 2147483649
+LongResult: 1281104301025214464
+LongResult: 1281104301025214467
```

Comment:
@IlyaEp Allowing this result makes the tests pass, I assume this is coming from the graalvm update